todoman/cli.py

```diff
@@ -9,6 +9,11 @@
 from .main import dump_idfile, get_task_sort_function, get_todo, get_todos
 from .model import Database, Todo
 from .ui import EditState, TodoEditor, TodoFormatter
+from .interactive import TodomanInteractive
+
+import logging
+logging.basicConfig()
```

Comment:
`basicConfig` should be called in the `cli` function, not at module level. In the case of showing a UI you won't want to output anything I think.